src/components/Admin/Toolbar.jsx

```diff
@@ -62,7 +64,36 @@ const Toolbar = ({
     );
     setToggle(!toggle);
   };
+  // const csvData = objects.entries(download).map(entry => rowData[entry[0]] = entry[1]);
+  // const csvData = objects;
+  const csvData = objects.map((download) => {
+    const rowData = {};
+
+    Object.entries(download).forEach(([key, value]) => {
+      if (key === "members" && Array.isArray(value) && value.length > 0) {
```

Comment:
don't make the assumption that it will be members always, we could have other fields that are arrays and it would work for that as well. u can simply check for the array type and length and then use the join functions